### DISCUSSION TITLE
Math block is not displayed correctly

### DISCUSSION THREAD

**Discussion by KYang-N** [Zettlr/Zettlr#3072] [Q&A] [answered]
Hi,
I entered an equation like this
![image](https://user-images.githubusercontent.com/37972210/150619328-c23a65d1-df03-4931-a83c-81a070c08468.png)
and I found that labels always overlap the formulae.
![image](https://user-images.githubusercontent.com/37972210/150619417-b548ba46-bc1e-40e5-97e8-66b1cd35d634.png)
Equations in \begin{align} \end{align} have the same problem.
When I tried to export the file in the format PDF (LaTeX), it failed and reported this error:
![image](https://user-images.githubusercontent.com/37972210/150619537-8a1bac19-de7f-4b58-a852-130464101a4f.png)
Thanks

**Comment by nathanlesage** [comment, marked as answer]
With regard to the labels, I noticed this myself. I currently suspect that this is because of how the elements are rendered, that the span is initially too narrow, triggering this overlap during rendering (maybe @tobiasdiez has some input…?).

With regard to the error: I think that Pandoc internally already replaces the double dollar signs with `\begin{equation}` and `\end{equation}`, which leads to the error. If you want to align the equation, you can use the environment `\begin{align}`.

**Reply by tobiasdiez** [reply]
The normal syntax would be without the extra `equation` block indeed.

**Reply by KYang-N** [reply]
I tried again and saw the point. When exporting in the PDF (LaTeX) format, the double dollar `$$` signs are replaced with `\begin{equation}
` and `\end{equation}`, so the `\begin{aligned}` environment works well but `\begin{align}` leads to an error. To label the equations in the PDF file, `\begin{align}` and `\end{align}` should be used without the double dollar signs, but then you won't be able to preview the formulae. 

Before switching to Zettlr,  I wrote with Typora. The `\begin{equation}` or `\begin{align}` can be used In its math blocks directly, and it won't cause errors when exported. I did not realize that it is not a universal feature.
